modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -208,6 +216,22 @@ components:
                     readOnly: true
                     type: object
             type: object
+        ValidateRequest:
+            properties:
+                validationKeys:
+                    items:
+                        type: string
+                    type: array
+                values:
+                    $ref: "#/components/schemas/ObjectEntry"
+        ValidateResult:
+            properties:
+                errors:
+                    items:
+                        $ref: "#/components/schemas/Error"
+                    type: array
+                success:
```

Comment:
Having `success` and `errors` is ambiguous. Maybe assuming that when there are no `errors` everything went well would be simpler.